tests/Behat/Page/Cart/SummaryPageInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\CustomerReorderPlugin\Behat\Page\Cart;
+
+use \Sylius\Behat\Page\Shop\Cart\SummaryPageInterface as BaseSummaryPageInterface;
+
+interface SummaryPageInterface extends BaseSummaryPageInterface
+{
+    public function checkout(): void;
+    public function countFlashMessages(): int;
+    public function assertFlashMessageWithTextExists(string $text): bool;
```

Comment:
If the method is named `assert...` I would expect it to not return anything, but throw an exception if assertion failed. Or we could name it `doesFlashMessageWithTextExist` 😄